assets/js/blocks/mini-cart/editor.scss

```
@@ -9,3 +9,9 @@
 .wc-block-editor-mini-cart__drawer-link {
 	margin-bottom: 24px;
 }
+
+.wc-block-editor-mini-cart-icon {
```

You may consider following the BEM convention as the rest of the file, however, I know it's not followed in the whole project, hence it's just a suggestion you may ignore.

```suggestion
.wc-block-editor-mini-cart__icon {
```